src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -25,11 +32,14 @@
  * Call {@link #startNewTest()} before each test to ensure a clean model.
  */
 class TestUtils {
+    protected Logger logger;
```

Comment:
You added the logger here, which I like.  But I think you should set the log level to INFO in the constructor here and remove that code from `declutterLogging()` as we talked about, and rename it to `initTestUtils()` or something like that.

This wouldn't prevent one of the test files to create their own logger instance (either the root or at the class level) with its own logLevel, even though right now none of the tests does that.

The point is that the TestUtils "automatically" declutter the logging a bit, but users of the utils can still override that behavior or log their own stuff independently.

Reply:
I tried your requested change but now I remember why I deferred this.  Moving the setting of the log level to TestUtils results in a bunch of Jena related DEBUG output from the logger that gets dumped before the log level is set.  I think that's because the call to ValidateAIF.createForLDCOntology method as the second parameter of the call to initialize the TestUtils generates it.

Reply:
The ValidateAIF main gets around this by setting the loglevel to INFO, but that isn't used by the unit tests.  The split of ExamplesAndValidationTest caused the line to set the level to be duplicated, and setting the log level in the TestUtil ctor is not enough.  The real solution is to use the logback configuration file(s) to set the log level properly for the unit tests.  That can be done in a separate ticket.

Reply:
Darn, you're right...it's even possible I found that during 718.  Agreed with your assessment.